### ISSUE TITLE
Test file missing for times.js

### ISSUE THREAD

**Issue by Deven1902** [100xdevs-cohort-2/assignments#110]
Testing suite is missing for the file medium/times.js file. Add a testing suite for the file.

**Comment by Kapil-2305** [comment]
I don't think we can create a accurate testing file for medium/time.js question. Because the output time for every computation is different every single time for everyone. Because it depends on the compiler.

In the below image you can clearly see that output is different for the same input.

<img width="896" alt="Screenshot 2023-12-06 at 12 30 38 PM" src="https://github.com/100xdevs-cohort-2/assignments/assets/120932175/42ce3478-a901-4a19-aa6d-ed3f3278ad2e">

<img width="899" alt="Screenshot 2023-12-06 at 12 30 52 PM" src="https://github.com/100xdevs-cohort-2/assignments/assets/120932175/0f0d069a-ffc5-427f-9a27-1f78075f75c8">

**Comment by V2K4Y** [comment]
> Testing suite is missing for the file medium/times.js file. Add a testing suite for the file.

Read the instructions carefully.
